app/src/main/java/hr/foi/air/servicesync/navigation/AppNavHost.kt

```diff
@@ -57,6 +56,13 @@ fun NavGraphBuilder.AppNavHost(navController: NavHostController) {
             companyName = companyName
         )
     }
+    composable("company/{companyName}/{serviceName}") {
```

Comment:
Nice. Continuing usage of proper implementation of navigation.

Reply:
Thanks!